06-ssb-read.js

```diff
@@ -1,33 +1,32 @@
-var SSB = require('ssb-db')
 var Paramap = require('pull-paramap')
 var pull = require('pull-stream')
-var db = SSB(null, {}, null, '/tmp/bench-ssb-legacy_ssb/')
-db.ready.set(true)
 
-var a = []
+var createSsb = require('secret-stack')(require('ssb-config'))
```

Comment:
would be nice not having to load ssb-config, as we don't want to listen on anything since that might clash with another instance of sbot already on the machine.